client/Profile.jsx

```diff
@@ -18,31 +18,16 @@ class InnerProfile extends React.Component {
             return;
         }
 
-        this.windowDefaults = {
-            plot: false,
-            draw: false,
-            challengeBegin: false,
-            attackersDeclared: true,
-            defendersDeclared: true,
-            dominance: false,
-            standing: false
-        };
-
-        this.timerSettingsDefaults = {
-            events: true,
-            abilities: false
-        };
-
         this.state = {
-            disableGravatar: this.props.user.settings ? this.props.user.settings.disableGravatar : false,
-            email: this.props.user ? this.props.user.email : '',
+            disableGravatar: this.props.user.settings.disableGravatar,
+            email: this.props.user.email,
```

Comment:
Nope, in fact it was already redundant, we bail further up if this.props.user is null